.github/workflows/Packages - EdFi.Ods.Extensions.SK.yml

```diff
@@ -0,0 +1,129 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Packages - EdFi.Ods.Extensions.Sk
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
To reduce clean up work later as part of ODS-5589, please remove the inputs here since they are not needed.

Reply:
I removed it . Thanks